lib/dev-server.js

```diff
@@ -120,6 +116,10 @@ async function devServer(options) {
         app.use(fallback(entryPoint, { root: options.contentBase }));
     }
 
+    if (options.after) {
+        options.after(app, server);
```

Comment:
Is there any particular reason for this to be moved below the fallback? This would be a breaking change, and it would probably cause the hook to not trigger at all if using historyApiFallback.